tt_eager/tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp

```diff
@@ -117,49 +117,55 @@ bool is_parametrized_type(T val) {
 
 struct UnaryWithParam {
```

Comment:
Should the name be updated to UnaryWithParams? Is that a plan for a future PR?

Reply:
I decided to rename it here, in a second commit.